R/transformCounts.R

```diff
@@ -117,21 +117,21 @@
 #'
 NULL
 
-#' @rdname transformAbundance
+#' @rdname transformCounts
 #' @export
-setGeneric("transformAbundance", signature = c("x"),
+setGeneric("transformCounts", signature = c("x"),
            function(x,
                     abund_values = "counts",
                     method = c("relabundance", "log10", "pa", "Z", "hellinger", "clr"),
                     name = method,
                     pseudocount = FALSE,
                     scalingFactor = 1)
-               standardGeneric("transformAbundance"))
+               standardGeneric("transformCounts"))
 
 
-#' @rdname transformAbundance
+#' @rdname transformCounts
 #' @export
-setMethod("transformAbundance", signature = c(x = "SummarizedExperiment"),
+setMethod("transformCounts", signature = c(x = "SummarizedExperiment"),
```

Comment:
should the `setMethods` in the codes also have `scalingFactor = 1` option.

Reply:
I think we previously decided to drop the `scale` transformation. But if there seems to be a clear use/need, then it would be good to add back. Is `scalingFactor` needed anywhere else?

Reply:
It was in the build warnings. Therefore, I highlighted it. If not needed then it should be removed. For specific cases the user can do the scaling by themselves.

Reply:
Ok, I will remove scalingFactor

Reply:
Removed